### ISSUE TITLE
tox-poetry-installer crashes due to import error

### ISSUE THREAD

**Issue by finswimmer** [enpaul/tox-poetry-installer#73]
Hey,

we released `poetry`1.2.0 and `poetry-core` 1.1.0 today :tada: Unfortunately this breaks `tox-poetry-installer` due to changed import paths.

Most of them can be fixed easily:

https://github.com/enpaul/tox-poetry-installer/blob/b3a5e869acf3fc63cb45bec42d34e36137f073b2/tox_poetry_installer/installer.py#L12

```py
from poetry.core.packages.package import Package as PoetryPackage 
```

and 

https://github.com/enpaul/tox-poetry-installer/blob/b3a5e869acf3fc63cb45bec42d34e36137f073b2/tox_poetry_installer/utilities.py#L13-L14

```py
from poetry.core.packages.dependency import Dependency as PoetryDependency
from poetry.core.packages.package import Package as PoetryPackage
```

But I'm  not sure what's the best way to fix https://github.com/enpaul/tox-poetry-installer/blob/b3a5e869acf3fc63cb45bec42d34e36137f073b2/tox_poetry_installer/_poetry.py#L33

This is not available anymore in poetry 1.2.0. Instead it's available in the most recent version of `cleo`:

```
from cleo.io.null_io import NullIO
```

While the easy changes are compatible with poetry < 1.2, the change of `NullIO` import isn't. Maybe just put another try/except in here to find put what's available?

fin swimmer

**Comment by enpaul** [comment]
Thanks for the report @finswimmer

I've just released version 0.8.5 which pins the Poetry requirement at less than 1.2 to back stop this issue in the immediate time frame. @Callek has a PR up to add support for Poetry 1.2 that I'll be looking at next to hopefully get this functionality added soon

**Comment by Callek** [comment]
> Thanks for the report @finswimmer
> 
> I've just released version 0.8.5 which pins the Poetry requirement at less than 1.2 to back stop this issue in the immediate time frame. @Callek has a PR up to add support for Poetry 1.2 that I'll be looking at next to hopefully get this functionality added soon

My pr pins to an alpha of poetry 1.2, but happy to bump that to the final release depending on Ethan's opinion here. 

Mostly just blocked on his time

**Comment by enpaul** [comment]
This issue should now be resolved by #70 , thanks to @Callek
